### PR TITLE
Using only 'uniqueId' as key

### PR DESCRIPTION
**Pull request by Avielyo10**

### CODE REVIEW

File: prometheus/cli.py [hunk 1]
```diff
@@ -55,9 +56,15 @@ def metrics(host, token, interval, time, skip_namespaces, output):
 
     keys = list(json.loads(d.replace('\'', '\"'))
                 for d in combined_metrics.keys())
+    [key.update({'uniqueId': ':'.join(map(str, key.values()))})
+     for key in keys]
+    keys = list({k: v for k, v in key.items() if k == 'uniqueId'}
+                for key in keys)
+
```
Comment (lack) [inline prometheus/cli.py]: Let's move this parsing into prometheus.Prometheus.Metric

**Review by lack — CHANGES_REQUESTED** [review]

**Comment by Avielyo10** [comment]
@lack hold

File: prometheus/cli.py (hunk 1 above)
Comment (Avielyo10) [reply prometheus/cli.py]: Ok, let's hold that since I found a bug, I'll fix it

Comment (Avielyo10) [reply prometheus/cli.py]: Fixed

**Review by lack — APPROVED** [review]
Awesome!  Much simpler!

**Pull request merged**